src/main/scala/nl/knaw/dans/api/sword2/DepositHandler.scala

```diff
@@ -62,11 +62,13 @@ object DepositHandler {
 
   def finalizeDeposit(mimeType: String)(implicit id: String): Try[Unit] = {
     log.info(s"[$id] Finalizing deposit")
+    baseDir = new File(SwordProps("basedir"))
```

Comment:
not sure what `val tempDir = new File(SwordProps("tempdir"), id)` represents, but shouldn't you do a similar check for this one?